src/classes/GE_GiftEntryController.cls

```diff
@@ -42,12 +42,12 @@ public with sharing class GE_GiftEntryController {
     private static GiftBatchService giftBatchService = new GiftBatchService();
 
     @TestVisible
-    private static ElevateCaptureGroupService captureGroupService {
+    private static ElevateBatchService elevateBatchService {
```

Comment:
@kenlewis-sfdo With this terminology shift, there are a couple places that feel like we are bleeding the elevate batch concept - which is specific to the Elevate adaptor - way too high up into our application. These references in GE_GiftEntryController feel like one instance, but curious to see if you see any good opportunities or approaches to improve this here!

Reply:
2 cents incoming, I think this is ok. Referencing the elevate batch service in the controller is more ideal than directly using the elevate batch domain and/or entity imo.

Something to discuss maybe is whether or not Elevate batches actually belong to a Gift Batch and so should these calls to the elevate batch service actually be behind the gift batch/gift batch service?

Reply:
I created this diagram to help us in that discussion - I think this is what we should refactor toward as this work continues to evolve: https://lucid.app/lucidchart/invitations/accept/inv_1d610dfc-bcb1-4ee9-8833-db0f6e52e552

Reply:
Thanks for sharing (and creating) that diagram, @kenlewis-sfdo! If I'm correctly understanding the vision, we would eventually want to avoid putting anything specific to Elevate into the UI layer - and so the use of the ElevateAuthorizedGift and ElevateTokenizedGift in the AuraEnabled methods of this controller would have to revisited?

Reply:
Yep, definitely.  The controller should be making calls to an NPSP service layer (something like "GiftEntryService") to add the Gift to a Gift Batch.  The Service layer should instantiate and coordinate among NPSP Domain classes to get work done.  

Once the GiftBatches are configured to use an instance of IPaymentServices (defining an `add` and a `process` method), then the *implementation* will use ElevateBatches etc.

So yes, ElevateTokenizedGift and ElevateAuthorizedGift should both be renamed and/or removed from the controller.